<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patient_data</class>
 <widget class="QDialog" name="patient_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patient Data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QLabel" name="temperature">
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="systolicBlPr">
     <property name="text">
      <string>Systolic bl. pr.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="name">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="genderLabel">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="medHistory">
     <property name="text">
      <string>Med History:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="heart_rate">
     <property name="maximum">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="med_history">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="systolic">
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="address">
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="patient_id">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="allergyLabel">
     <property name="text">
      <string>Allergy:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="diastolicBlPr">
     <property name="text">
      <string>Diastolic bl. pr.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="phone_number">
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="birthDate">
     <property name="text">
      <string>Birth Date</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="diastolic"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="patientID">
     <property name="text">
      <string>Patient ID</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="allergy">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="gender">
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="birth_date">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="heartRate">
     <property name="text">
      <string>Heart Rate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="phoneNumber">
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="temp">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>301.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>patient_data</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
